### ISSUE TITLE
Forbidden URL in configure yaml

### ISSUE THREAD

**Issue by ericdoi** [gjwgit/azcv#3]
The `https://farm4.staticflickr.com/3883/15144849957_f326e03f75_b.jpg` demo image seems to throw an error when accessing from the script (thought it's fine from my browser).  Maybe we can make these demo files optional so the config doesn't fail?

```
$ ml configure azcv

*** The following required pip packages are already installed:
  azure-cognitiveservices-vision-computervision

*** Downloading required files ...
...trying url https://mlhub.ai/empty.txt

    * https://mlhub.ai/empty.txt
      using cached copy found in /Users/ericdoi/.mlhub/azcv/private.txt ...
...trying url https://github.com/gjwgit/azcv/raw/master/images/mycat.png

    * https://github.com/gjwgit/azcv/raw/master/images/mycat.png
      using cached copy found in /Users/ericdoi/.mlhub/azcv/cache/images/mycat.png ...
...trying url https://raw.githubusercontent.com/Azure-Samples/cognitive-services-sample-data-files/master/ComputerVision/Images/landmark.jpg

    * https://raw.githubusercontent.com/Azure-Samples/cognitive-services-sample-data-files/master/ComputerVision/Images/landmark.jpg
      using cached copy found in /Users/ericdoi/.mlhub/azcv/cache/images/landmark.jpg ...
...trying url http://cdn1.thr.com/sites/default/files/2013/11/marina_bay_sands_singapore_a_l.jpg

    * http://cdn1.thr.com/sites/default/files/2013/11/marina_bay_sands_singapore_a_l.jpg
      using cached copy found in /Users/ericdoi/.mlhub/azcv/cache/images/marina_bay_sands_singapore.jpg ...
...trying url https://www.wayoutback.com.au/assets/Uploads/Uluru.jpg

    * https://www.wayoutback.com.au/assets/Uploads/Uluru.jpg
      using cached copy found in /Users/ericdoi/.mlhub/azcv/cache/images/uluru.jpg ...
...trying url https://res.cloudinary.com/uktv/image/upload/v1402498133/mztwkl0sze26zjpc0xdh.jpg

    * https://res.cloudinary.com/uktv/image/upload/v1402498133/mztwkl0sze26zjpc0xdh.jpg
      using cached copy found in /Users/ericdoi/.mlhub/azcv/cache/images/celebrities.jpg ...
...trying url https://raw.githubusercontent.com/Azure-Samples/cognitive-services-sample-data-files/master/ComputerVision/Images/objects.jpg

    * https://raw.githubusercontent.com/Azure-Samples/cognitive-services-sample-data-files/master/ComputerVision/Images/objects.jpg
      using cached copy found in /Users/ericdoi/.mlhub/azcv/cache/images/objects.jpg ...
...trying url https://farm4.staticflickr.com/3883/15144849957_f326e03f75_b.jpg
Traceback (most recent call last):
  File "/Users/ericdoi/miniconda3/envs/bevsight/bin/ml", line 8, in <module>
    sys.exit(main())
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/site-packages/mlhub/__init__.py", line 197, in main
    args.func(args)
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/site-packages/mlhub/commands.py", line 897, in configure_model
    utils.install_file_deps(deplist, model, key=args.i, yes=YES)
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/site-packages/mlhub/utils.py", line 1177, in install_file_deps
    filename = get_url_filename(
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/site-packages/mlhub/utils.py", line 377, in get_url_filename
    info = urllib.request.urlopen(url).getheader("Content-Disposition")
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/Users/ericdoi/miniconda3/envs/bevsight/lib/python3.8/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by gjwgit** [comment]
Thanks for reporting the issue. 

The problem arises because the python request was not setting headers suitably. 

A fix will be available soon.

A workaround is to download the file manually into the local cache and then reconfigure:

```
$ wget https://farm4.staticflickr.com/3883/15144849957_f326e03f75_b.jpg -o ~/.mlhub/.cache/azcv/cache/images/streetscape.jpg
$ ml configure azcv
```

**Comment by gjwgit** [comment]
Resolved with mlhub version 3.5.20
